ajax/libs/drift-zoom/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "drift-zoom",
+  "description": "Easily add \"zoom on hover\" functionality to your site's images. Lightweight, no-dependency JavaScript.",
+  "filename": "Drift.js",
```

```suggestion
  "filename": "Drift.min.js",
```
Would it not be better to suggest the minified file by default?

---

Agreed 👍